From 604b61173405a5a278abf5f85c8d3dc9e9f682b4 Mon Sep 17 00:00:00 2001
From: yvanliang <yvan6liang@outlook.com>
Date: Fri, 9 Oct 2026 15:43:37 +0800
Subject: [PATCH 06/23] GSI Android 17 compatibility:
 0006-Ugly-but-secure-Set-dev-uinput-as-0666-to-fix-finger.patch

Change-Id: Id919ae87a0296d3a0660136b41b60e0fd8f0adc4
---
 rootdir/ueventd.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rootdir/ueventd.rc b/rootdir/ueventd.rc
index 80ef6a1..1da9707 100644
--- a/rootdir/ueventd.rc
+++ b/rootdir/ueventd.rc
@@ -64,7 +64,7 @@ subsystem vfio
 
 # these should not be world writable
 /dev/uhid                 0660   uhid       uhid
-/dev/uinput               0660   uhid       uhid
+/dev/uinput               0666   uhid       uhid
 /dev/rtc0                 0640   system     system
 /dev/tty0                 0660   root       system
 /dev/graphics/*           0660   root       graphics
